bw2io/extractors/simapro_csv.py

```diff
@@ -272,6 +363,25 @@ def get_global_parameters(cls, data, pm):
 
     @classmethod
     def get_project_name(cls, data):
+        """
+        Extract the project name from the given data.
+
+        Parameters
+        ----------
+        data : list
+            A list of data, where each item is a list of strings representing a row of the data.
+
+        Returns
+        -------
+        str
+            The project name.
+
+        Notes
+        -----
+        This method searches for a row in the data where the first item starts with "{Project:" or "{Projet:".
```

Comment:
Should this not go directly below the short description at the beginning of the docstring? Why is it under 'Notes'?

Reply:
Moved.